### PR TITLE
Milestone 07: Diagram Based

### PR DESCRIPTION
**Pull request by mavericks-db**
## Diagram Based

**This branch includes:**

- [x] Take a look at the following database diagram for a clinic:
clinic_diagram
- [x] Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- [x] Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- [x] Remember to add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram/schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE DATABASE clinic;
+
```
Comment (ShadyShawkat) [inline schema_based_on_diagram/schema_based_on_diagram.sql:4]: - [ ] Kindly implement the relationship table between `medical_histories` and `treatments` tables and apply their foreign keys.
- [ ] Please make sure to implement indices for all of the foreign keys.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @mavericks-db, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mavericks-db** [comment]
Thank you 👍

File: schema_based_on_diagram/schema_based_on_diagram.sql (hunk 1 above)
Comment (mavericks-db) [reply schema_based_on_diagram/schema_based_on_diagram.sql]: Done on commits https://github.com/mavericks-db/vet-clinic-database/pull/7/commits/f63baa90301bec15ca996a49ea1ee357158666bc and https://github.com/mavericks-db/vet-clinic-database/pull/7/commits/d5fd2bec5e2dd4587ef196ffb8d71508b1964727

**Pull request merged**